src/hisp/festim_models/mb_model.py

```diff
@@ -277,267 +542,206 @@ def make_W_mb_model(
-#         surface_reaction_dd,
-#         surface_reaction_dt,
-#         surface_reaction_tt,
-#     ]
-
-#     ############# Exports #############
-
-#     my_model.exports = [
-#         F.VTXSpeciesExport(f"{folder}/mobile_concentration_t.bp", field=mobile_T),
-#         F.VTXSpeciesExport(f"{folder}/mobile_concentration_d.bp", field=mobile_D),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_d1.bp", field=trap1_D),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_t1.bp", field=trap1_T),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_d2.bp", field=trap2_D),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_t2.bp", field=trap2_T),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_d3.bp", field=trap3_D),
-#         F.VTXSpeciesExport(f"{folder}/trapped_concentration_t3.bp", field=trap3_T),
-#     ]
-
-#     quantities = {}
-#     for species in my_model.species:
-#         quantity = F.TotalVolume(field=species, volume=w_subdomain)
-#         my_model.exports.append(quantity)
-#         quantities[species.name] = quantity
-
-#     ############# Settings #############
-#     my_model.settings = F.Settings(
-#         atol=1e-15,
-#         rtol=1e-15,
-#         max_iterations=1000,
-#         final_time=final_time,
-#     )
-
-#     my_model.settings.stepsize = F.Stepsize(initial_value=1)
-
-#     return my_model, quantities
\ No newline at end of file
+
+def make_DFW_mb_model(
+    temperature: Callable | float | int,
+    deuterium_ion_flux: Callable,
+    tritium_ion_flux: Callable,
+    deuterium_atom_flux: Callable,
+    tritium_atom_flux: Callable,
+    final_time: float,
+    folder: str,
+    L: float,
+) -> Tuple[CustomProblem, Dict[str, F.TotalVolume]]:
+    """Create a FESTIM model for the DFW MB scenario.
+
+    Args:
+        temperature: the temperature in K.
+        deuterium_ion_flux: the deuterium ion flux in m^-2 s^-1.
+        tritium_ion_flux: the tritium ion flux in m^-2 s^-1.
+        deuterium_atom_flux: the deuterium atom flux in m^-2 s^-1.
+        tritium_atom_flux: the tritium atom flux in m^-2 s^-1.
+        final_time: the final time in s.
+        folder: the folder to save the results.
+        L: the length of the domain in m.
+
+    Returns:
+        the FESTIM model, the quantities to export.
+    """
+    my_model = CustomProblem()
+
+    ############# Material Parameters #############
+
+    L = L  # m
+    vertices = np.concatenate(  # 1D mesh with extra refinement
+        [
+            np.linspace(0, 30e-9, num=200),
+            np.linspace(30e-9, 3e-6, num=300),
+            np.linspace(3e-6, 30e-6, num=200),
+            np.linspace(30e-6, L, num=200),
+        ]
+    )
+    my_model.mesh = F.Mesh1D(vertices)
+
+    # DFW material parameters
+    dfw_density = 8.45e28  # atoms/m3
+    D_0 = 1.45e-6  # m^2/s
+    E_D = 0.59  # eV
+    dfw = F.Material(
+        D_0=D_0,
+        E_D=E_D,
+        name="dfw",
+    )
```

Comment:
1. DFW is not a material, is it steel? tungsten?
2. where do these numbers come from? can they be pulled from HTM?

Reply:
these are ITER's 'mean parameters' between Kaeri values and "Penzhorn" values -- taking mean values i think is an ITER norm

Reply:
That's ok, but in that case you need to write it/comment it.

But still, what material is this? is it steel? is it tungsten? is it something else?

Reply:
Btw KAERI is an experiment, not a reference. The same materials could be measured several times in the same experiment but in different conditions and give different results